unicon_backend/workers/consumer.py

```diff
@@ -27,13 +36,53 @@ def message_callback(
         response: JobResult = JobResult.model_validate_json(body)
         with SessionLocal() as db_session:
             task_result = db_session.scalar(
-                select(TaskResultORM).where(TaskResultORM.job_id == str(response.id))
+                sa.select(TaskResultORM).where(col(TaskResultORM.job_id) == response.id)
```

Comment:
is it better to have `TaskResultORM::job_Id` be a `Mapped` attribute instead of using `col` here? we could probably standardise the use of `col` or `Mapped` across the board

Reply:
when i use Mapped on a primitive type:, mypy says no errors, but pydantic will start throwing errors when the api is called

so I have been using
- `col` for primitives
- `Mapped` for relationships

```
File "/Users/seelengxd/Desktop/projects/unicon/unicon_backend/.venv/lib/python3.12/site-packages/pydantic/_internal/_generate_schema.py", line 515, in _unknown_type_schema
    raise PydanticSchemaGenerationError(
pydantic.errors.PydanticSchemaGenerationError: Unable to generate pydantic-core schema for sqlalchemy.orm.base.Mapped[str | None]. Set `arbitrary_types_allowed=True` in the model_config to ignore this error or implement `__get_pydantic_core_schema__` on your type to fully support it.

If you got this error by calling handler(<some type>) within `__get_pydantic_core_schema__` then you likely need to call `handler.generate_schema(<some type>)` since we do not call `__get_pydantic_core_schema__` on `<some type>` otherwise to avoid infinite recursion
```